codex-ui/dev/pages/components/Fieldset.vue

```diff
@@ -5,10 +5,40 @@
       One or more fields grouped together with a title
     </template>
   </PageHeader>
+  <Fieldset
+    title="Fieldset title"
+  >
+    <Section title="section title">
+      <Row
+        label="label"
+        title="actual row title"
+        subtle="subtle"
+      >
+        <template #right>
+          <Button icon="Plus" />
+        </template>
+      </Row>
+    </Section>
+    <Section title="section title">
+      <Row
+        label="label"
+        title="actual row title"
+        subtle="subtle"
+      >
+        <template #right>
+          <Button icon="Plus" />
+        </template>
+      </Row>
+    </Section>
+    <Button>
+      button
```

Comment:
Button for what?

Reply:
this is just a demonstration file, there could be any other component